scanner/src/components/DataRow/index.svelte

```diff
@@ -1,37 +1,57 @@
 <script lang="ts">
   export let label: string
   export let value: string | string[]
+  export let link: boolean = false
+  export let column: boolean = false
 </script>
 
-<div class="row">
+<div class={`data ${column ? "column" : "row"}`}>
   <div class="label">
     {label}:
   </div>
-  <div class="value">
-    {#if typeof value === "string"}
-      {value.trim() || "-"}
-    {:else if value === null || value === undefined}
-      -
-    {:else if value.length === 0}
-      <div>-</div>
-    {:else}
-      {#each value as v}
-        <div>{v.trim() || "-"}</div>
-      {/each}
-    {/if}
-  </div>
+  {#if link || (typeof value === "string" && value.startsWith("https://"))}
+    <div class="value">
+      {#if typeof value === "string"}
+        <a href={value} target="_blank" rel="noreferrer">{value}</a>
+      {:else if value === null || value === undefined || value.length === 0}
+        -
+      {:else}
+        {#each value as v}
+          <a href={v} target="_blank" rel="noreferrer">{v}</a>
+        {/each}
+      {/if}
+    </div>
+  {:else}
+    <div class="value">
+      {#if typeof value === "string"}
+        {value.trim() || "-"}
+      {:else if value === null || value === undefined || value.length === 0}
+        -
+      {:else}
+        {#each value as v}
+          <div>{v.trim() || "-"}</div>
+        {/each}
+      {/if}
+    </div>
+  {/if}
 </div>
 
 <style lang="scss">
-  .row {
+  .data {
     display: flex;
-    flex-direction: row;
     font-size: 1rem;
     font-style: normal;
     font-weight: 400;
     &:not(:last-child) {
       margin-bottom: 1rem;
     }
+    &.row {
+      flex-direction: row;
+      align-items: center;
+    }
+    &.column {
+      flex-direction: column;
+    }
   }
```

Comment:
Looks like this code is missing formatting, there's supposed to be empty lines between the blocks. Ensure this is fixed everywhere.

```suggestion
    font-weight: 400;
    
    &:not(:last-child) {
      margin-bottom: 1rem;
    }
    
    &.row {
      flex-direction: row;
      align-items: center;
    }
    
    &.column {
      flex-direction: column;
    }
  }
```